rust/connlib/tunnel/src/lib.rs

```diff
@@ -347,6 +349,18 @@ where
     }
 
     fn poll_next_event(&mut self, cx: &mut Context<'_>) -> Poll<Event<TId>> {
+        if self.stats_timer.poll_tick(cx).is_ready() {
+            let (node_stats, conn_stats) = self.node.stats();
+
+            tracing::info!(target: "connlib::stats", "{node_stats:?}");
+
+            for (id, stats) in conn_stats {
+                tracing::info!(target: "connlib::stats", %id, "{stats:?}");
+            }
```

Comment:
I made a dedicated log target for this. I think in the log-run, we'll need to be more specific about our log targets. Currently, configuring the log string is a mess because we depend on the internals on what the crates are called a lot.